doc/INTERNAL.rst

```diff
@@ -200,27 +200,28 @@ show that Pythran can compile it::
 Assignment and AugAssignment statements at top-level are one of the most common
 cases of Python programs::
 
-    $> printf 'a = 1 + (2 + 2) * 3\nprint a\na += 1\nprint a' > internal_top_level_assign.py
+    $> printf 'a = 1 + (2 + 2) * 3\nprint a' > internal_top_level_assign.py
     $> pythran internal_top_level_assign.py
     $> python -c 'import internal_top_level_assign'
     13
-    14
 
 Then let's test Python's flow control keywords: for, while, if-else:
 
 Loop statement at top-level::
 
-    $> printf 's=0\nfor i in range(10):s+=i\nprint s' > internal_top_level_loop.py
+    $> printf 's=0\nfor i in range(2): print s' > internal_top_level_loop.py
     $> pythran internal_top_level_loop.py
     $> python -c 'import internal_top_level_loop'
-    45
+    0
+    0
 
 While statement at top-level::
 
-    $> printf 'i=0\nwhile i<10:i+=1\nprint i' > internal_top_level_while.py
+    $> printf 'i=0\nwhile True:\n print i\n break\nprint i' > internal_top_level_while.py
     $> pythran internal_top_level_while.py
     $> python -c 'import internal_top_level_while'
-    10
+    0
+    0
```

Comment:
> Because i += 1 is a modification on globals which is not allowed anymore.
ok. Should we put this in a functiondef instead of top-level, for the
sake of the doc?